etc/infrasim.full.yml.example

```diff
@@ -0,0 +1,79 @@
+# Unique identifier
+name: node-1
```

Comment:
@xiar @fub2 @sharkconi this is a full configuration example, which can be used in this version and start esxi.